src/composer.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "eliksir/mailmojo-wp-widget",
+  "description": "MailMojo WordPress Plugin",
+  "keywords": [
+    "wordpress",
+    "plugin",
+    "mailmojo",
+    "newsletter",
+    "mailing list",
+    "email"
+  ],
+  "homepage": "https://mailmojo.no",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Stian Prestholdt",
+      "email": "stian@e5r.no",
+      "homepage": "http://e5r.no"
+    }
+  ],
+  "type": "wordpress-plugin",
+  "require": {
+    "php": ">=5.3.2",
+    "composer/installers": "v1.0.6",
```

Comment:
I'm not sure but most people don't have a `v` here so I've left it out since we tag our releases explicitly just with version number.